test/Microsoft.Azure.SignalR.Management.Tests/JwtTokenHelper.cs

```diff
@@ -0,0 +1,68 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.IdentityModel.Tokens.Jwt;
+using System.Linq;
+using System.Security.Claims;
+using System.Text;
+using Microsoft.IdentityModel.Tokens;
+
+namespace Microsoft.Azure.SignalR.Management.Tests
+{
+    internal static class JwtTokenHelper
```

Comment:
this is used by multiple test projects, consider reuse existing one?

Reply:
Add a project `Microsoft.Azure.SignalR.TestUtility` in `test` folder and put methods to reuse them?